app/src/main/cpp/skyline/services/codec/IHardwareOpusDecoder.cpp

```diff
@@ -0,0 +1,126 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright © 2021 Skyline Team and Contributors (https://github.com/skyline-emu/)
+
+#include "IHardwareOpusDecoder.h"
+#include "results.h"
+
+namespace skyline::service::codec {
+
+    size_t CalculateOutBufferSize(i32 sampleRate, i32 channelCount, i32 frameSize) {
```

Comment:
```suggestion
    static size_t CalculateOutBufferSize(i32 sampleRate, i32 channelCount, i32 frameSize) {
```

Reply:
this is also used in `IHardwareOpusDecoderManager.cpp` so cannot be made static

Reply:
ah fair enough then